packages/app-bridge/src/types/Document.ts

```diff
@@ -84,7 +85,7 @@ export type DocumentApi = Simplify<
             change_comment: Nullable<string>;
             change_comment_by: Nullable<string>;
             change_title: Nullable<string>;
-            targets: any;
+            targets: Nullable<SingleTargetApi[]>;
```

Comment:
This is correct, but also you need to do remapping for the `Document` type to `targets: number[]`. As we want to store only IDs.